point-addition/src/conversion.rs

```diff
@@ -0,0 +1,91 @@
+//! This module implements the conversion from the sum of two elliptic curve points to the sum of
+//! two field elements of the field underlying the elliptic curve
+
+use super::{PointAddition, PointAdditionError};
+use async_trait::async_trait;
+use p256::EncodedPoint;
+use share_conversion_aio::{AdditiveToMultiplicative, MultiplicativeToAdditive};
+use share_conversion_core::fields::{p256::P256, Field};
+
+/// The instance used for converting the curve points
+pub struct Converter<T, U, V>
+where
+    T: AdditiveToMultiplicative<V>,
+    U: MultiplicativeToAdditive<V>,
+    V: Field,
+{
+    /// Performs conversion of addition to multiplication
```

Comment:
Here and below "conversion" is correct, since we do A2M conversion for the field elements